recipe/meta.yaml

```diff
@@ -19,12 +19,13 @@ requirements:
 
   run:
     - python >=3.7
-    - bokeh >=1.0.0,!=2.0.0
-    - cytoolz >=0.8.2
     - dask-core {{ version }}
     - distributed {{ version }}
+    - cytoolz >=0.8.2
     - numpy >=1.18
     - pandas >=1.0
+    - bokeh >=1.0.0,!=2.0.0
+    - jinja2
```

Comment:
Thanks @jakirkham! `jinja2` is included as a dependency for `distributed`. Just for my own understanding, is there a reason to include it here as well?

Reply:
It is. Though it looks like Jacob added this to Dask’s extra requirements ( https://github.com/dask/dask/pull/8019 ). Even though it is in `distributed`, it is better to have it here in case `distributed` were ever to drop it (just like we include `bokeh` here). IOW explicit is better than implicit 🙂